utils/dataloaders.py

```diff
@@ -782,12 +786,13 @@ def load_image(self, i):
                 im = cv2.imread(f)  # BGR
                 assert im is not None, f'Image Not Found {f}'
             h0, w0 = im.shape[:2]  # orig hw
+            im_orig = np.copy(im)
             r = self.img_size / max(h0, w0)  # ratio
             if r != 1:  # if sizes are not equal
                 interp = cv2.INTER_LINEAR if (self.augment or r > 1) else cv2.INTER_AREA
                 im = cv2.resize(im, (math.ceil(w0 * r), math.ceil(h0 * r)), interpolation=interp)
-            return im, (h0, w0), im.shape[:2]  # im, hw_original, hw_resized
-        return self.ims[i], self.im_hw0[i], self.im_hw[i]  # im, hw_original, hw_resized
+            return im, (h0, w0), im.shape[:2], im_orig  # im, hw_original, hw_resized
```

Comment:
I don't really get these comments repeating what's already written in code, but anyway, `im_orig` is missing.

Reply:
ill remove them